compiler/Makefile

```diff
@@ -8,12 +9,18 @@ BAL_TEST_FLAGS ?= $(BAL_BUILD_FLAGS) --disable-groups longRunning
 
 all: $(COMPILER_JAR)
 
+pack: $(COMPILER_PACK)
+
 test:
 	$(BAL) test $(BAL_TEST_FLAGS)
 
 $(COMPILER_JAR): $(COMPILER_SRC)
 	$(BAL) build --target-dir ../build $(BAL_BUILD_FLAGS)
 
+$(COMPILER_PACK): $(COMPILER_SRC)
+	$(BAL) pack --target-dir ../build $(BAL_BUILD_FLAGS)
+	$(BAL) push $(COMPILER_PACK) --repository=local
```

Comment:
Doesn't push accept a `--target-dir` option? If not, please create issue.

Reply:
created issue https://github.com/ballerina-platform/ballerina-lang/issues/37163